### PR TITLE
Feature/OR 138 - Rebrand of Disclosure

### PR DESCRIPTION
**Pull request by fraincs**
Issue:

## Summary

Disclosure component needs to be branded with the Workleap theme.

## What I did

- Replaced Orbit tokens with the Hopper ones.

### CODE REVIEW

File: packages/components/src/disclosure/src/Disclosure.css [hunk 1]
```diff
@@ -15,9 +15,9 @@
 
 /* CONTENT */
 .o-ui-disclosure-content-inner {
-    padding-top: var(--o-ui-sp-5);
-    padding-bottom: var(--o-ui-sp-5);
-    color: var(--o-ui-text-alias-primary);
+    padding-top: var(--hop-space-inset-md);
```
Comment (vicky-comeau) [inline packages/components/src/disclosure/src/Disclosure.css:18]: are we not doing the style like we see in figma? 
image

Comment (alexasselin008) [reply packages/components/src/disclosure/src/Disclosure.css:18]: that would be a "bordered" accordion component to orbit: 
https://orbit.sharegate.design/?path=/docs/accordion--default-story#bordered

Comment (vicky-comeau) [reply packages/components/src/disclosure/src/Disclosure.css:18]: Did you already talk to design about this? That seems to be a different component that yes would probably use the same styling.

Comment (vicky-comeau) [reply packages/components/src/disclosure/src/Disclosure.css:18]: I suppose it would end up being a big change for orbit. It's just a bit confusing.

**Review by alexasselin008 — APPROVED** [review]

**Pull request merged**